libcperciva/datastruct/elasticarray.c

```diff
@@ -28,6 +28,12 @@ resize(struct elasticarray * EA, size_t nsize)
 	if (EA->alloc < nsize) {
 		/* We need to enlarge the buffer. */
 		nalloc = EA->alloc * 2;
+		/*
```

Comment:
Almost there!  Just a few nitpicks left.

In addition to two missing commas in the comment, there's a few things in the commit message.  I'm going to copy the original one, so that it's easier to see in the github web interface:

> This commit adds a code comment describing the cases handled by the if (nalloc < nsize) block. One such case is the potential wrap around of EA->alloc * 2. Without the code comment, it was not quite apparent upon seeing the code that the wrap around is properly handled.

- Please use two spaces after a period.  (Yes, it's a somewhat niche style of English writing, but that's what it says in `STYLE`.)

- I would delete "code" in "code comment" (this happens twice)
- I would replace "quite apparent upon seeing the code" with "clear".

  So that part of the sentence would be "...it was not clear that the wrap..."